templates/layouts/configs/_footer.html

```diff
@@ -1,7 +1,12 @@
+{% set contact_us_url = "https://census.gov.uk/contact-us/" %}
+{% set cookies_url = "https://census.gov.uk/cookies/" %}
+{% set help_url = "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/" %}
```

Comment:
Can we just make this part of an `else` just so it is not potentially set twice?

```jinja
{% if schema_theme == "census-nisra" %}

  {% set contact_us_url = "https://census.gov.uk/ni/contact-us/" %}
  {% set cookies_url = "https://census.gov.uk/ni/cookies/" %}
  {% set help_url = "https://census.gov.uk/ni/help/how-to-answer-questions/online-questions-help/" %}
  {% set logo = "nisra-logo-black-en" %}
  {% set alt = "Nisra Logo" %}
  {% set class_small_logo = "footer__poweredby-img--small" %}

{% elif language_code == "cy" %}

  {% set contact_us_url = "https://cyfrifiad.gov.uk/cysylltu-a-ni/" %}
  {% set cookies_url = "https://cyfrifiad.gov.uk/cwcis/" %}
  {% set help_url = "https://cyfrifiad.gov.uk/help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein-ew/" %}
  
{% else %}
  
  {% set contact_us_url = "https://census.gov.uk/contact-us/" %}
  {% set cookies_url = "https://census.gov.uk/cookies/" %}
  {% set help_url = "https://census.gov.uk/help/how-to-answer-questions/online-questions-help/" %}

{% endif %}
```

Reply:
So, originally I had it like that above but I had a comment on another PR from Ian who suggested changing it to how it is now. Ill leave it as it is for now, unless anyone else has a problem with it and we can change it in another PR